tests/integration/test_xrdsst.py

```diff
@@ -0,0 +1,134 @@
+
+import unittest
+import time
+import subprocess
+import os
+import json
+import git
+import urllib3
+
+from definitions import ROOT_DIR
+from xrdsst.controllers.base import BaseController
+from xrdsst.controllers.init import InitServerController
+from xrdsst.controllers.timestamp import TimestampController
+from xrdsst.controllers.token import TokenController
+from xrdsst.main import XRDSSTTest
+
+
+class TestXRDSST(unittest.TestCase):
+    git_repo = 'https://github.com/nordic-institute/X-Road.git'
+    local_folder = os.path.join(ROOT_DIR, "tests/integration/X-Road")
+    branch_name = 'develop'
+    docker_folder = local_folder + '/Docker/securityserver'
+    dist = 'bionic-6.25.0'
+    image = 'xroad-security-server:' + dist
+    url = 'https://localhost:4000/api/v1/api-keys'
+    roles = '[\"XROAD_SYSTEM_ADMINISTRATOR\",\"XROAD_SECURITY_OFFICER\"]'
+    header = 'Content-Type: application/json'
+    file_name = 'api-key.txt'
+    max_retries = 100
+    curl_retry_wait_seconds = 5
+
+    api_key = None
+
+    @staticmethod
+    def single_ss_config():
+        configuration_anchor = os.path.join(ROOT_DIR, "tests/resources/configuration-anchor.xml")
+        config = {
+            'logging': [{'file': '/var/log/xrdsst_test.log', 'level': 'INFO'}],
+            'security-server':
+                [{'name': 'ss',
+                  'url': 'https://localhost:8000/api/v1',
+                  'api_key': 'X-Road-apikey token=a2e9dea1-de53-4ebc-a750-6be6461d91f0',
+                  'configuration_anchor': configuration_anchor,
+                  'owner_dn_country': 'EE',
+                  'owner_dn_org': 'ORG',
+                  'owner_member_class': 'GOV',
+                  'owner_member_code': '1234',
+                  'security_server_code': 'SS',
+                  'software_token_id': 0,
+                  'software_token_pin': '1234'}]}
+        return config
+
+    def setUp(self):
```

Comment:
Check if python libraries for docker test running allow simpler definition of this running of the containers.

Reply:
now the docker-related operations make use of Docker SDK, should be more clear